include/ts/apidefs.h.in

```diff
@@ -461,6 +468,7 @@ typedef enum {
   TS_EVENT_LIFECYCLE_SERVER_SSL_CTX_INITIALIZED = 60021,
   TS_EVENT_LIFECYCLE_CLIENT_SSL_CTX_INITIALIZED = 60022,
   TS_EVENT_LIFECYCLE_TASK_THREADS_READY         = 60027,
+  TS_EVENT_LIFECYCLE_SHUTDOWN                   = 60028,
```

Comment:
It's a little wonky that we're not keeping these in numerical order, but by "type". Pros and cons, but I think I'd prefer them in numerical order. Can be done as a separate PR though, since it involves changing others (but please don't forget).

Reply:
It's better as a separate PR because that PR will be incompatible. For now, you could backport this one.